langchain/agents/agent_toolkits/prompt.ts

```diff
@@ -0,0 +1,75 @@
+export const SQL_PREFIX = `You are an agent designed to interact with a SQL database.
```

Comment:
probably nice to split these up? not super scalable as we start to add more

maybe structure can be

json
  - agent
  - toolkit
  - prompt
sql
   - agent
   - toolkit
   - prompt

etc